block/manager.go

```diff
@@ -145,23 +144,28 @@ func (m *Manager) Start(ctx context.Context) error {
 	}
 
 	if !isSequencer {
+		// Fullnode loop can start before syncing from DA
 		go uevent.MustSubscribe(ctx, m.Pubsub, "applyGossipedBlocksLoop", p2p.EventQueryNewNewGossipedBlock, m.onNewGossipedBlock, m.logger)
 	}
 
+	// TODO: populate the accumulatedSize on startup
+
+	// Start syncing from DA
```

Comment:
useless comment, if `syncBlockManager` is hard to understand than rename the func instead

Reply:
I mean just `// Start syncing from DA`

Reply:
removed